sushichef.py

```diff
@@ -755,6 +762,14 @@ def download(source_id):
     return False
 
 
+def get_video_resolution_format(video, resolution="720", ext="mp4"):
```

Comment:
Could you change `resolution` argument to something like `maxvres` or `maxvertres` (maximum vertical resolution) and return the first stream with vertical resolution smaller than or equal to `maxvres`.

I think asking for a specific 480 or 720 should work for most videos since youtube generates them, but you never know it might not be available for all so better get whatever we have up to `maxvres`.